src/scrapers/twitter_batch.iced

```diff
@@ -0,0 +1,107 @@
+{BaseScraper,BaseBearerToken} = require './base'
+{TwitterScraper} = require './twitter'
+{constants} = require '../constants'
+{v_codes} = constants
+{Lock} = require '../util'
+urlmod = require 'url'
+
+#================================================================================
+
+ws_normalize = (x) ->
+  v = x.split(/[\t\r\n ]+/)
+  v.shift() if v.length and v[0].length is 0
+  v.pop() if v.length and v[-1...][0].length is 0
+  v.join ' '
+
+#================================================================================
+
+exports.TwitterBatchScraper = class TwitterBatchScraper extends TwitterScraper
+  constructor: (opts) ->
+    @_tweet_cache = opts.tweet_cache
+    @cache_refresh_interval = opts.cache_refresh_interval
+    super opts
+
+  _hunt_batch : (cb) ->
+    # Make a query to find all keybase proofs since `last_id` (if present).
+    query =
+      query : "\"Verifying myself\" \"Keybase.io\""
+      expansions: "author_screen_name"
+      "user.fields": "url,username"
+      "tweet.fields": "created_at"
+      max_results: 60
```

Comment:
Might be problematic during initial run if there were more than new 60 proofs last week.